src/components/ui/css/MainPanel.css

```diff
@@ -119,15 +119,15 @@
 }
 
 .cmap{
-    padding: 1px 0px;
+    padding: 2px 0px;
     cursor: pointer;
     max-width: 200px;
     height: 24px;
-    border-radius: 4px;
+    border-radius: 6px;
 }
 
 .cmap:hover{
-    transform: scale(1.25, 0.9);
+    transform: scale(1.05, 1.4);

```

Comment:
no, that was intensional, the effect looked nicer. But we can come back later to the whole hover effect approach.